tests/transformers/test_transformers_model_export.py

```diff
@@ -3640,7 +3640,7 @@ def test_whisper_model_supports_timestamps(raw_audio_file, whisper_pipeline):
         model_info = mlflow.transformers.log_model(
             transformers_model=whisper_pipeline,
             artifact_path="model",
-            inference_config=inference_config,
+            model_config=model_config,
```

Comment:
Let's revert changes in this file, and add other tests for 
1. saving/logging model with load_config works fine
2. pass extra kwargs in mlflow.pyfunc.load_model will update the configs saved, and applied during loading

For model serving and other flavors support, I can update in follow-up PRs after we get this one  merged in :)